<?xml version="1.0" encoding="UTF-8"?>
<com.ibm.datatools.etl.errmsgs:Message xmi:version="2.0" xmlns:xmi="http://www.omg.org/XMI" xmlns:com.ibm.datatools.etl.errmsgs="http:///com/ibm/datatools/etl/errmsgs.ecore" msgNum="12209" timestamp="1548318453424" msgTxt="Validation succeeded for the 'LAB2Q1A' flow. However, there were warnings." msgSeverity="WARN" component="dataflow.validation" msgDescTxt="Validation succeeded for LAB2Q1A with warnings." msgCause="" msgResponse="" entityType="DataFlow">
  <expandedMsgs msgNum="12208" timestamp="1548318453127" msgTxt="Validation succeeded for the 'flowLabel'." msgSeverity="LOG" component="dataflow.validation" msgDescTxt="Validation succeeded for flowLabel." msgCause="" msgResponse="" entityType="DataFlow" operatorID="01"/>
  <expandedMsgs msgNum="12208" timestamp="1548318453424" msgTxt="Validation succeeded for the 'flowLabel'." msgSeverity="LOG" component="dataflow.validation" msgDescTxt="Validation succeeded for flowLabel." msgCause="" msgResponse="" entityType="DataFlow" operatorID="02"/>
  <expandedMsgs msgNum="12208" timestamp="1548318453424" msgTxt="Validation succeeded for the 'flowLabel'." msgSeverity="LOG" component="dataflow.validation" msgDescTxt="Validation succeeded for flowLabel." msgCause="" msgResponse="" entityType="DataFlow" operatorID="04">
    <expandedMsgs msgNum="13775" timestamp="1548318453424" msgTxt="Insert : LAB2Q1A_RESULT : Potentially invalid column mapping between 'VARCHAR(32) SNAME' of INPUT_04_0' and 'VARCHAR(20) SNAME' of 'LAB2Q1A_RESULT'" msgSeverity="WARN" component="cflow.core" msgDescTxt="Bad column type mapping." msgCause="While iterating through the map, found that the 'to' and 'from' types might be incompatible." msgResponse="Use equivalent and valid column types. " entityType="CONTROLFLOW" operatorID="04"/>
  </expandedMsgs>
</com.ibm.datatools.etl.errmsgs:Message>
